### PR TITLE
Machine Learning Extension / ModelDB

### PR DESCRIPTION
**Pull request by oluies**
https://mitdbg.github.io/modeldb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ A curated list of awesome [Apache Spark](https://spark.apache.org/) packages and
 * [KeystoneML](http://keystone-ml.org/) - Type safe machine learning pipelines with RDDs.
 * [JPMML-Spark](https://github.com/jpmml/jpmml-spark) - PMML transformer library for Spark ML.
 * [Distributed Keras](https://github.com/cerndb/dist-keras) - Distributed deep learning framework with PySpark and Keras.
+* [ModelDB](https://mitdbg.github.io/modeldb) - A system to manage machine learning models
```
Comment (zero323) [inline README.md]: - Missing full stop at the and of the line.
- Since project is not Spark specific I would like to see a bit more precise description.

Comment (oluies) [inline README.md]: Found this video https://www.youtube.com/watch?v=gxBb4CjJcxQ&feature=youtu.be
Maybe you can submit a Python 3 upgrade

**Comment by eliasah** [comment]
I'll take it for a test !

**Comment by eliasah** [comment]
Ok so no python 3 support and someone maybe needs to dockerize their env to make it testable...

File: README.md [hunk 2]
```diff
@@ -92,6 +92,7 @@ A curated list of awesome [Apache Spark](https://spark.apache.org/) packages and
 * [KeystoneML](http://keystone-ml.org/) - Type safe machine learning pipelines with RDDs.
 * [JPMML-Spark](https://github.com/jpmml/jpmml-spark) - PMML transformer library for Spark ML.
 * [Distributed Keras](https://github.com/cerndb/dist-keras) - Distributed deep learning framework with PySpark and Keras.
+* [ModelDB](https://mitdbg.github.io/modeldb) - A system to manage machine learning models for spark.ml and scikit-learn
```
Comment (zero323) [inline README.md]: Still missing a full stop :)

**Pull request merged**